config/ant-phase-verify.xml

```diff
@@ -66,8 +66,10 @@
         <fileset dir="${basedir}" includes="**/*">
           <exclude name=".git/**/*"/>
           <exclude name="src/main/**/*"/>
+          <exclude name="src/xdocs/**/*"/>
+          <exclude name="src/site/**/*"/>
```

Comment:
are we sure that we will not miss valiation ob other files except for https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_checks.xml#L23

Reply:
I don't think we're missing anything. https://github.com/checkstyle/checkstyle/issues/8104#issuecomment-614530791

Reply:
As we touched this .. .lets configs do not overlap at all.
-----
checkstyle_checks.xml
  "src/"" exclude all resource folders
  "fileExtensions"="java, properties, xml, vm, g, g4, dtd"

checkstyle_resources_checks.xml
  "src/" with only resources
  "fileExtensions" is not defined - all files

checkstyle_non_main_files_checks.xml
  "." all file and folders except for "src" 

-----

so lets exclude whole "src" folder from main.
Please recheck that Checks we do not lose any Check, if required ... we can copy Checks to/from config.

Reply:
```checkstyle_checks``` is checking just ```src``` and excludes all ```resources```. How can we exclude ```src``` from it?

Reply:
```checkstyle_resources``` overlaps with ```checkstyle_checks``` for a couple of checks. However, ```checkstyle_non_main_files_checks``` is a subset of ```checkstyle_checks```. I don't think we loose any check. How should I go about removing this overlap? Honestly speaking, I don't think it is necessary to do so as they are checking different files.

Reply:
>  How can we exclude src from it?

can we use multiple `fileset`s ?  https://stackoverflow.com/q/25992260/1015848

> checkstyle_resources overlaps with checkstyle_checks for a couple of checks.

it is ok, as they are working on different folders.

----

as we make `checkstyle_non_main_files_checks` to not look at src folder , we need to make sure that `checkstyle_resources` and `checkstyle_checks` do have same Checks in them.

Reply:
> as we make checkstyle_non_main_files_checks to not look at src folder , we need to make sure that checkstyle_resources and checkstyle_checks do have same Checks in them.

Done!

Reply:
> we can copy Checks to/from config.

Done! Copied the relevant suppressions too.